<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextLayerDialog</class>
 <widget class="QWidget" name="TextLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Text layer settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Text editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="StyledFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFontComboBox" name="fontEdit">
               <property name="toolTip">
                <string>Font</string>
               </property>
               <property name="currentFont">
                <font/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="fontSizeEdit">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Font size</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="boldBtn">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Bold</string>
               </property>
               <property name="text">
                <string>Bold</string>
               </property>
               <property name="icon">
                <iconset resource="../MishiraApp.qrc">
                 <normaloff>:/Resources/wysiwyg-bold.png</normaloff>:/Resources/wysiwyg-bold.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="italicBtn">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Italic</string>
               </property>
               <property name="text">
                <string>Italic</string>
               </property>
               <property name="icon">
                <iconset resource="../MishiraApp.qrc">
                 <normaloff>:/Resources/wysiwyg-italic.png</normaloff>:/Resources/wysiwyg-italic.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="underlineBtn">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Underline</string>
               </property>
               <property name="text">
                <string>Underline</string>
               </property>
               <property name="icon">
                <iconset resource="../MishiraApp.qrc">
                 <normaloff>:/Resources/wysiwyg-underline.png</normaloff>:/Resources/wysiwyg-underline.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="leftBtn">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Align text left</string>
                  </property>
                  <property name="text">
                   <string>Align text left</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../MishiraApp.qrc">
                    <normaloff>:/Resources/wysiwyg-left.png</normaloff>:/Resources/wysiwyg-left.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="centerBtn">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Align text center</string>
                  </property>
                  <property name="text">
                   <string>Align text center</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../MishiraApp.qrc">
                    <normaloff>:/Resources/wysiwyg-center.png</normaloff>:/Resources/wysiwyg-center.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="rightBtn">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Align text right</string>
                  </property>
                  <property name="text">
                   <string>Align text right</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../MishiraApp.qrc">
                    <normaloff>:/Resources/wysiwyg-right.png</normaloff>:/Resources/wysiwyg-right.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="justifyBtn">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Justify</string>
                  </property>
                  <property name="text">
                   <string>Justify</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../MishiraApp.qrc">
                    <normaloff>:/Resources/wysiwyg-justify.png</normaloff>:/Resources/wysiwyg-justify.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ColorButton" name="colorBtn">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Text color</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Background:</string>
               </property>
               <property name="buddy">
                <cstring>bgColorBtn</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ColorToggleButton" name="bgColorBtn">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Background color</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="documentEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Text options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="StyledFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Stroke size:</string>
            </property>
            <property name="buddy">
             <cstring>strokeSizeEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Stroke color:</string>
            </property>
            <property name="buddy">
             <cstring>strokeColorBtn</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="UnitLineEdit" name="strokeSizeEdit">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="ColorButton" name="strokeColorBtn">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="StyledFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Word wrapping:</string>
            </property>
            <property name="buddy">
             <cstring>wordWrapBtn</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="wordWrapBtn">
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="noWordWrapBtn">
               <property name="text">
                <string>Disabled</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="StyledFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Horizontal scroll speed:</string>
            </property>
            <property name="buddy">
             <cstring>xScrollEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="UnitLineEdit" name="xScrollEdit"/>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>363</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Vertical scroll speed:</string>
            </property>
            <property name="buddy">
             <cstring>yScrollEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="UnitLineEdit" name="yScrollEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StyledFrame</class>
   <extends>QFrame</extends>
   <header>Widgets/styledframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>Widgets/colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>UnitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/unitlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ColorToggleButton</class>
   <extends>QPushButton</extends>
   <header>Widgets/colortogglebutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>fontEdit</tabstop>
  <tabstop>fontSizeEdit</tabstop>
  <tabstop>boldBtn</tabstop>
  <tabstop>italicBtn</tabstop>
  <tabstop>underlineBtn</tabstop>
  <tabstop>leftBtn</tabstop>
  <tabstop>centerBtn</tabstop>
  <tabstop>rightBtn</tabstop>
  <tabstop>justifyBtn</tabstop>
  <tabstop>colorBtn</tabstop>
  <tabstop>bgColorBtn</tabstop>
  <tabstop>documentEdit</tabstop>
  <tabstop>strokeSizeEdit</tabstop>
  <tabstop>strokeColorBtn</tabstop>
  <tabstop>wordWrapBtn</tabstop>
  <tabstop>noWordWrapBtn</tabstop>
  <tabstop>xScrollEdit</tabstop>
  <tabstop>yScrollEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../MishiraApp.qrc"/>
 </resources>
 <connections/>
</ui>
